### PR TITLE
Client th schedule many refreshrs

### PR DESCRIPTION
**Pull request by timh1203**
*normal priority*
*pull --> no conflicts*

-Add logic to schedule 2d, 2wk, 2mo from input date
-Add render of the dates on page for user's ease
-Add some basic styles
-Code to save to DB is commented out for now

### CODE REVIEW

File: client/src/components/classesPage/ClassCreateView.js [hunk 1]
```diff
@@ -133,20 +135,122 @@ function ClassCreateView(props) {
...
+          console.log(`219`);
+          return addRefreshr(newRefreshr);
+        }
+      })
+      .then(res => {
+        if (res.status === 201) {
+          validated.campaign_id.push(res.data.id);
+          console.log(`226`);
+          console.log(timeTriData[2]);
+          console.log(validated.campaign_id[2]);
```
Comment (szincone) [inline client/src/components/classesPage/ClassCreateView.js]: Still tons of console.logs but we don't need to remove them until next week.

File: client/src/components/classesPage/ClassCreateView.js [hunk 2]
```diff
@@ -133,20 +135,122 @@ function ClassCreateView(props) {
...
+      })
+      .then(res => {
+        if (res.status === 201) {
+          validated.schedule_code = res.status;
+          console.log(
+            `Success! Your campaign ${res.data.id} is scheduled for ${
+            res.data.send_at
+            }. Status is "${res.data.status}"!`
+          );
+          // setTimeout(() => {
```
Comment (szincone) [inline client/src/components/classesPage/ClassCreateView.js]: Same deal w/ housekeeping unused comments.

File: client/src/components/classesPage/forms/CampaignForm.js [hunk 3]
```diff
@@ -102,12 +102,27 @@ const styles = theme => ({
...
+  }
 });
 
 function CampaignForm(props) {
   const [refreshrs, setRefreshrs] = useState([]);
   const [activeRefreshr, setActiveRefreshr] = useState(null);
 
+  // For displaying on the page for the ease of user
+  // current time, 2 days, 2 weeks, 2 months
+  let [schedule, setSchedule] = useState({
```
Comment (szincone) [inline client/src/components/classesPage/forms/CampaignForm.js]: Great job w/ these and getting them to display.

File: client/src/components/classesPage/forms/CampaignForm.js [hunk 4]
```diff
@@ -182,26 +196,37 @@ function CampaignForm(props) {
...
   const alterTime = (e) => {
     e.preventDefault()
-    // tacking time onto campaign data for submitClassData()
-    props.setCampaignData({
-      ...props.campaignData,
-      date: e.target.value
-    });
+    const schedule0 = moment(`${e.target.value}T00:00:00`).format('ddd, MMMM Do, YYYY ha')
+    const schedule1 = moment(`${e.target.value}T00:00:00`).add(2, 'day').format('ddd, MMMM Do, YYYY ha')
+    const schedule2 = moment(`${e.target.value}T00:00:00`).add(2, 'weeks').format('ddd, MMMM Do, YYYY ha')
```
Comment (szincone) [inline client/src/components/classesPage/forms/CampaignForm.js]: Good job figuring out momentjs.

File: client/src/components/classesPage/forms/CampaignForm.js [hunk 5]
```diff
@@ -221,12 +246,41 @@ function CampaignForm(props) {
             defaultValue={today}
             onChange={e => alterTime(e)}
           />
+          <div className={classes.scheduleDiv}>
```
Comment (szincone) [inline client/src/components/classesPage/forms/CampaignForm.js]: If you wanted to convert the whole thing to material-ui, pretty sure you can use `Grid` for divs, looks really good w/o too though.

File: client/src/components/classesPage/ClassCreateView.js (hunk 1 above)
Comment (timh1203) [reply client/src/components/classesPage/ClassCreateView.js]: Yea, was planning to after we square away with saving to the DB.

File: client/src/components/classesPage/forms/CampaignForm.js (hunk 5 above)
Comment (timh1203) [reply client/src/components/classesPage/forms/CampaignForm.js]: Will try that.

**Pull request merged**